packages/odyssey-react/src/components/Box/_box-other.scss

```diff
@@ -33,3 +33,16 @@
 .userSelectAll {
   user-select: all;
 }
+
+/* Element Resets */
+
+.asFieldset {
+  margin-block: 0;
+  margin-inline: 0;
+  padding-block: 0;
+  padding-inline: 0;
+  border: 0;
+  font: inherit;
```

Comment:
I *think* these two font properties might be dangerous since we can't control the source order of these flat classes. @andrewberg-okta you've done the most investigation here, I think.

Reply:
Good call.